docs/modules/ROOT/pages/vshn-managed/postgresql/usage.adoc

```diff
@@ -5,32 +5,176 @@ The latter references the `postgres-creds` secret that will contain the access k
+
+```
+# example assumes reuse of names and namespace from example above
+kubectl -n app1-prod describe vshnpostgresqls.vshn.appcat.vshn.io postgresql-prod 
+```
+
+Example output looks as follows:
+
+```
+Status:
+  Certificate Debug:
+    Last Transition Time:  2023-02-13T15:11:44Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:42Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  Conditions:
+    Last Transition Time:  2023-02-13T15:11:39Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+    Last Transition Time:  2023-02-13T15:12:21Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+  Connection Details:
+    Last Published Time:  2023-02-13T15:12:21Z
+  Local CA Debug:
+    Last Transition Time:  2023-02-13T15:11:43Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:41Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  Namespace Debug:
+    Last Transition Time:  2023-02-13T15:11:41Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:41Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  Network Policy Debug:
+    Last Transition Time:  2023-02-13T15:11:45Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:43Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  Pgcluster Debug:
+    Last Transition Time:  2023-02-13T15:11:48Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:47Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  Pgconfig Debug:
+    Last Transition Time:  2023-02-13T15:11:44Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:43Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  Profile Debug:
+    Last Transition Time:  2023-02-13T15:11:44Z
+    Reason:                Available
+    Status:                True
+    Type:                  Ready
+    Last Transition Time:  2023-02-13T15:11:42Z
+    Reason:                ReconcileSuccess
+    Status:                True
+    Type:                  Synced
+  s3BackupConfigDebug:
```

Comment:
Is this by intention that this is named differently? All other status fields with "Debug" in the name do have an uppercase first letter and a space before "Debug".

Reply:
this is probably just formatting from `kubectl describe` it doesn't want to split words when it see number in name